src/graphics-private.h

```diff
@@ -105,8 +105,10 @@ typedef struct _Graphics {
 	cairo_t			*ct;
 	GpMatrix		*copy_of_ctm;
 	cairo_matrix_t		previous_matrix;
+#ifdef CAIRO_HAS_XLIB_SURFACE
 	Display			*display;
```

Comment:
It looks like Display and Drawable are never used in the codebase (except for setting them to NULL), so I'd guess it'd be better to just remove them alltogether?